pyproject.toml

```diff
@@ -32,7 +32,7 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = ">=3.7.0,<4.0"
+python = ">=3.8.0,<4.0"
```

Comment:
Yes sure 👍🏾. Earlier, I removed Python 3.7 because poetry install was breaking since the pytest-xprocess package had an issue with 3.7 but I just checked and everything seems fine now.